astroquery/besancon/tests/test_besancon.py

```diff
@@ -55,40 +54,36 @@ def test_reader(filename, length, ncols, d1, mv1):
 @pytest.fixture
 def patch_post(request):
     mp = request.getfuncargvalue("monkeypatch")
-    mp.setattr(requests, 'post', post_mockreturn)
+    mp.setattr(besancon.Besancon, '_request', post_mockreturn)
     return mp
 
 
 @pytest.fixture
 def patch_get_readable_fileobj(request):
     @contextmanager
     def get_readable_fileobj_mockreturn(filename, **kwargs):
-        # file_obj = StringIO.StringIO(filename)
         if isinstance(filename, string_types):
-            is_binary = kwargs.get('encoding', None) == 'binary'
-            file_obj = open(data_path(filename),
-                            "r" + ('b' if is_binary else ''))
+            if '1376235131.430670' in filename:
+                is_binary = kwargs.get('encoding', None) == 'binary'
+                file_obj = open(data_path('1376235131.430670.resu'),
+                                "r" + ('b' if is_binary else ''))
         else:
             file_obj = filename
-        # file_obj = data_path(filename)
         yield file_obj
     mp = request.getfuncargvalue("monkeypatch")
     mp.setattr(commons, 'get_readable_fileobj',
                get_readable_fileobj_mockreturn)
     return mp
 
 
-def post_mockreturn(url, data, timeout=10, stream=True, params=None, **kwargs):
-    # filename = data_path('1376235131.430670.resu')
+def post_mockreturn(method, url, data, timeout=10, stream=True, **kwargs):
     filename = data_path('query_return.iframe.html')
     content = open(filename, 'rb').read()
     return MockResponseBesancon(content, filename, **kwargs)
 
 
 def test_query(patch_post, patch_get_readable_fileobj):
-    B = besancon.Besancon()
-    B.url_download = ''
-    result = B.query(0, 0, 'adam.g.ginsburg@gmail.com')
+    result = besancon.Besancon.query(0, 0, 'adam.g.ginsburg@gmail.com')
```

Comment:
we should make this e-mail address configurable so that individuals can run their own tests.... but for now, it's fine.  It worked, btw.